androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/SharedPreferencesTokenStore.java

```diff
@@ -0,0 +1,87 @@
+// Copyright 2019 Google Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.androidbrowserhelper.trusted;
+
+import android.content.Context;
+import android.content.SharedPreferences;
+import android.util.Base64;
+
+import androidx.annotation.Nullable;
+import androidx.browser.trusted.Token;
+import androidx.browser.trusted.TokenStore;
+
+/**
+ * Implements a {@link TokenStore} that uses {@link SharedPreferences} as the storage mechanistm
+ * for the {@link Token}.
+ */
+public class SharedPreferencesTokenStore implements TokenStore {
+    private static final String SHARED_PREFERENCES_NAME = "com.google.androidbrowserhelper";
+    private static final String KEY_TOKEN =
```

Comment:
The token is going to be stored in a Preferences file called "com.google.androidbrowserhelper", so I think the first half of `KEY_TOKEN` is redundant.

(But up to you, I'm not too fussed.)

Reply:
done